pkg/validator/types.go

```diff
@@ -54,6 +55,11 @@ const (
 	modifiedProperty = "The %s property is read-only."
 	modifiedRole     = "Role(%s) properties other than the members count cannot be modified while role members exist."
 
+	clusterNotReady      = "The %s cluster cannot be upgraded, as currently it is in %s state."
```

Comment:
We should probably have different messages for the 'cluster can't be upgraded because it is busy with other changes' case vs. the 'cluster is already upgrading and can only be rolled back to the previous app' case.

Reply:
Done